src/main/java/seedu/foodrem/model/item/ItemDate.java

```diff
@@ -0,0 +1,73 @@
+package seedu.foodrem.model.item;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an item date in an {@link Item}.
+ * Guarantees: details are present and not null, immutable.
+ */
+public class ItemDate {
```

Comment:
Would this be better as an abstract class to prevent instantiation?

Reply:
Roger